src/olympia/lib/crypto/signing.py

```diff
@@ -130,7 +130,7 @@ def autograph_sign_data(file_obj):
     signed_manifest = six.text_type(jar.signatures)
 
     signing_request = [{
-        'input': b64encode(force_bytes(signed_manifest)),
+        'input': force_text(b64encode(force_bytes(signed_manifest))),
```

Comment:
Just double checked, this is indeed what we want/need here.

Reply:
Yeah. It was broken because the contents weren't `str`, so that was failing to be encoded in json.

Reply:
Also had to do this in the other function in https://github.com/mozilla/addons-server/pull/10598/commits/1bee2d718984369645ba1868add692413fbea051 (locally this was working, which makes it difficult to track)